sites/ademe/src/backend/templates/richie/styleguide/index.html

```diff
@@ -0,0 +1,331 @@
+{% extends "richie/base.html" %}
```

Comment:
It's a pity that we need to override this whole template :scream:  Can you explain why you override it and what we can change in richie to allow minimal overrides?

Reply:
Oh yeah, don't bother of it, this a temporary change for some needed debugging information with new color palette for Ademe. It was planned to be removed once i finished (which i realize it should be now).

There is definitively no reason for sites from factory to override the styleguide template.